sdk/communication/communication-sms/test/smsClient.spec.ts

```diff
@@ -34,70 +36,132 @@ describe("SmsClient", async () => {
     }
   });
 
-  it("sends a SMS message to multiple recipients", async () => {
-    const connectionString = env.AZURE_COMMUNICATION_LIVETEST_CONNECTION_STRING as string;
+  //helper functions
+  const expectCorrectResult = (actualSmsResult: SmsSendResult, expectedRecipient: string) => {
+    assert.equal(actualSmsResult.httpStatusCode, 202);
+    assert.equal(actualSmsResult.to, expectedRecipient);
+    assert.isString(actualSmsResult.messageId);
+    assert.isTrue(actualSmsResult.successful);
+    assert.notExists(actualSmsResult.errorMessage, "no error message for success");
+  };
+
+  const expectIncorrectResult = (actualSmsResult: SmsSendResult, expectedRecipient: string) => {
+    assert.equal(actualSmsResult.httpStatusCode, 400);
+    assert.equal(actualSmsResult.to, expectedRecipient);
+    assert.notExists(actualSmsResult.messageId, "no message id for errors");
+    assert.isFalse(actualSmsResult.successful);
+    assert.exists(actualSmsResult.errorMessage);
```

Comment:
Should we verify the specific error message?

Reply:
hmm...yeah actually we can